tensorflow/python/ops/image_ops_impl.py

```diff
@@ -4491,13 +4492,36 @@ def ssim_multiscale(img1,
     A tensor containing an MS-SSIM value for each image in batch.  The values
     are in range [0, 1].  Returns a tensor with shape:
     broadcast(img1.shape[:-3], img2.shape[:-3]).
+
+  Raises:
+    ValueError: When static shape check fails.
```

Comment:
This specific error type is only returned in graph mode.  In eager, it will likely be an `InvalidArgumentError`.

Reply:
Hi @cantonios , thank you! How do you suggest me to specify this error? I'll add another error if running in graph mode. Is it makes sense?

Reply:
Looks like most of our existing docs just say `ValueError`, as you have it now.  I guess it's fine as-is.